2024-12-09 19:55:39,898 | WARNING | Неверный тип данных для объекта Runner
2024-12-09 19:55:39,901 | WARNING | Traceback (most recent call last):
  File "C:\PycharmProjects\lesson2\.venv\less12\tests_12_4.py", line 25, in test_run
    runn = ru_tour.Runner(12, 10)
  File "C:\PycharmProjects\lesson2\.venv\less12\rt_with_exceptions.py", line 6, in __init__
    raise TypeError(f'Имя может быть только строкой, передано {type(name).__name__}')
TypeError: Имя может быть только строкой, передано int

2024-12-09 19:55:39,901 | WARNING | Неверная скорость для Runner
2024-12-09 19:55:39,902 | WARNING | Traceback (most recent call last):
  File "C:\PycharmProjects\lesson2\.venv\less12\tests_12_4.py", line 14, in test_walk
    wal = ru_tour.Runner('walk', -5)
  File "C:\PycharmProjects\lesson2\.venv\less12\rt_with_exceptions.py", line 11, in __init__
    raise ValueError(f'Скорость не может быть отрицательной, сейчас {speed}')
ValueError: Скорость не может быть отрицательной, сейчас -5

